test/test_onnx.py

```diff
@@ -68,21 +70,38 @@ def forward(self, boxes, scores):
 
         self.run_model(Module(), (boxes, scores))
 
-    def test_roi_pool(self):
+    def test_roi_align(self):
         x = torch.rand(1, 1, 10, 10, dtype=torch.float32)
         single_roi = torch.tensor([[0, 0, 0, 4, 4]], dtype=torch.float32)
         model = ops.RoIAlign((5, 5), 1, 2)
         self.run_model(model, (x, single_roi))
 
-    def test_roi_align(self):
+    def test_roi_pool(self):
         x = torch.rand(1, 1, 10, 10, dtype=torch.float32)
         rois = torch.tensor([[0, 0, 0, 4, 4]], dtype=torch.float32)
         pool_h = 5
         pool_w = 5
         model = ops.RoIPool((pool_h, pool_w), 2)
-        model.eval()
         self.run_model(model, (x, rois))
 
+    def test_multi_scale_roi_align(self):
+
+        class TransformModule(torch.nn.Module):
+            def __init__(self):
+                super(TransformModule, self).__init__()
+                self.model = ops.MultiScaleRoIAlign(['feat1', 'feat2'], 3, 2)
+                self.image_sizes = [(512, 512)]
+
+            def forward(self, input, boxes):
+                return self.model(input, boxes, self.image_sizes)
+
+        i = OrderedDict()
+        i['feat1'] = torch.rand(1, 5, 64, 64)
+        i['feat2'] = torch.rand(1, 5, 16, 16)
+        boxes = torch.rand(6, 4) * 256
+        boxes[:, 2:] += boxes[:, :2]
+        self.run_model(TransformModule(), (i, [boxes],))
```

Comment:
Do you think it would be good to have a test that runs the same traced model for two different configurations of boxes (with different number of boxes), to make sure that the traced model works fine under different configurations?

Reply:
Couldn't add a test with different number of boxes since boxes is an input of this test, and input lists are flattened, which will result in a variant number of inputs, and we only support a fixed number of inputs.
But I will cover this case by adding a test for an end-to-end model (maybe faster_rcnn)  with 2 different input images that results in a different number of boxes.